src/components/editor/ResourceTemplate.jsx

```diff
@@ -27,11 +27,12 @@ class ResourceTemplate extends Component {
     if (!this.props.resourceTemplate) {
       return errorMessage
     }
+
     return (
       <div className="ResourceTemplate">
         <div id="resourceTemplate" style={{ marginTop: '-30px' }}>
           <section className="col-md-9">
-            <h1><em>{this.props.title}</em></h1>
+            <h1><em>{this.props.resourceTemplateLabel}</em></h1>
```

Comment:
we changed this from "title" in this context, because inside the resourceTemplate, it is "resourceLabel".  It is a "title" of sorts from the perspective of the html we generate.

Reply:
Also, the props are now mapped from the redux state, and not passed down as `title` from the parent component.

Reply:
👍